Pipfile

```
@@ -4,6 +4,11 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
+ruff = "==0.0.267"
```

Don't version pin them here, that will happen automatically in the lock file.  If you pin it here then the auto-updates will never find anything newer.  Just use this across the board
```suggestion
ruff = "*"
```

---

Done